### PR TITLE
I removed a couple of unused and/or <div/> like components

### PR DESCRIPTION
**Pull request by olafur-palsson**

### CODE REVIEW

File: app/App.js [hunk 1]
```diff
@@ -224,7 +224,7 @@ class App extends React.Component {
 
   renderFooter () {
     return (
-      <Footer className={styles.footer}>
+      <div className={styles.footer}>
```
Comment (nukeop) [inline app/App.js]: Better to use the footer tag here for semantics

File: app/App.js [hunk 2]
```diff
@@ -218,13 +218,13 @@ class App extends React.Component {
...
         </a>
-      </div>
+      </footer>
     );
   }
 
   renderFooter () {
     return (
-      <div className={styles.footer}>
+      <footer className={styles.footer}>
```
Comment (olafur-palsson) [inline app/App.js]: I also changed the 'sidebar_footer' for consistency

File: app/App.js (hunk 1 above)
Comment (olafur-palsson) [reply app/App.js]: Done.

**Comment by nukeop** [comment]
There is one change that I don't understand:
 app/components/Footer/styles.css → app/containers/wrapComponent.js 

Shouldn't it just be deleted?

**Pull request closed without merging**